containers/payments/Payment.js

```diff
@@ -22,7 +23,8 @@ const Payment = ({
     method,
     onMethod,
     onValidCard,
-    onPay
+    onPay,
+    fluidDisplay
```

Comment:
I prefer if you define `fieldClassName` prop (`String`).

Reply:
Okay, done: https://github.com/ProtonMail/react-components/pull/266/commits/b8eb830b7129fc6dfc70a4bad7ecc31d9e4c2bb3